src/components/SingleChoiceListItem.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useEffect, useState } from 'react'
+import { View } from 'react-native'
+import { capitalizeFirstLetter, getArticleColor } from '../utils/helpers'
+import { ARTICLES } from '../constants/data'
+import { COLORS } from '../constants/colors'
+import styled from 'styled-components/native'
+
+const StyledContainer = styled.View`
+  height: 55px;
```

Comment:
Do not like this height solution. Achieving the same result with padding would be nicer.

Reply:
As this creates problems with the StyledOverlay I created a new issue in which the styling should be generally improved.